.kitchen.dokken.yml

```diff
@@ -56,12 +56,16 @@ platforms:
     image: dokken/centos-7
     privileged: true
     pid_one_command: /usr/lib/systemd/systemd
+    internediate_instructions:
```

Comment:
Should this be spelt `intermediate_instructions` instead?

Reply:
Yep, fancy making a PR?

Reply:
Coming right up.